examples/upbound-aws-provider/composite-resources/database-examples/aurora/README.md

```diff
@@ -0,0 +1,174 @@
+
+NAME                          XR-KIND   XR-APIVERSION                 AGE
+xauroras.db.awsblueprint.io   XAurora   db.awsblueprint.io/v1alpha1   5m
+
+
+If we have about same kind of output as above we can apply th claim: ( We are using a namespace as team-a in this example, if you want to change , please go ahead and create and update the same in the claim)
+
+ ```shell
+cd ../composite-resources/database-examples/aurora
+k apply -f aurora-postgresql.yaml
+```
+
+We can check the execution of claim by following command:
+
+```shell
+k get Aurora -n team-a
+k describe Aurora -n team-a
+```
+(It should take about 15-20 min to provision the Aurora RDS cluster.)
+Below is the default behaviour of the resource which will be provisioned through the claim, just to mention all this default behaviour can be overridden through patching .
+
+ 1. It will create a 3 node cluster with 1 writer and 2 reader endpoint spread across 3 Azs by default.
+ 2. It will also create a proxy to connect to the aurora cluster.
+ 3. The management of database credential is done through the secret manager.
+ 4. The security group has 2 rules , one to allow the app to connect to the proxy and one within the security group
+     where the proxy can connect to the database (Make sure you have provided the correct the CIDR at the claim which is the allow CIDR on the security group from app.)
+ 5. The Aurora DB has been configured with logging and monitoring.
+
+### Steps to check the connectivity from a pod 
+Create an  policy to allow the pod to connect to the Aurora Database
+  
+```shell
+aws iam create-policy \
+--policy-name rdsproxy-access \
+--policy-document file://rdsproxy-access.json
+```
+Create a service account passing the above policy arn
+
+```shell
+POLICY_ARN=$(aws iam list-policies --query 'Policies[?PolicyName==`rdsproxy-access`].Arn' --output text)
+CLUSTER_NAME=$(aws eks list-clusters --query 'clusters[]' --output text | grep <YOUR-CLUSTER-NAME>)
+
+eksctl create iamserviceaccount \
+--name rds-access \
+--namespace team-a \
+--cluster ${CLUSTER_NAME} \
+--attach-policy-arn ${POLICY_ARN} \
+--approve \
+-override-existing-serviceaccounts
+``` 
+Generate a DB token which can be used for password when asked
+
+```shell
+PROXY_TERGET-ENDPOINT=k get secrets aurora-cluster-secrets  -n team-a -o json | jq -r .data.proxyEndpoint | base64 --decode
+CLUSTER_USER_NAME=k get secrets aurora-cluster-secrets  -n team-a -o json | jq -r .data.clusterUsername | base64 --decode
+
+
+aws rds generate-db-auth-token \
+--hostname ${PROXY_TERGET-ENDPOINT}  \
+--port 5432 \
+--region ${REGION} \
+--username ${CLUSTER_USER_NAME}
+```
+create the pod with psql client:
+
+```shell
+k apply -f psql-client-pod.yaml
+```
+
+Exec into the pod
+
+```shell
+k exec -it postgres-client -n team-a  -- sh
+```
+
+create a DB connection and provide the token which you generate earlier for password.
+
+```shell
+psql -h ${PROXY_TERGET-ENDPOINT}  -U ${CLUSTER_USER_NAME} -d aurorapgsqldb -W
```

Comment:
replace variable names

Reply:
replaced the variable name as it would be available in the pod.